src/inference/src/dev/iremote_tensor.cpp

```diff
@@ -62,7 +62,7 @@ class BlobTensor : public ITensor {
     }
 
     const Shape& get_shape() const override {
-        m_shape = blob->getTensorDesc().getBlockingDesc().getBlockDims();
+        m_shape = blob->getTensorDesc().getDims();
```

Comment:
It is incorrect change. So OpenVINO tensors don't have layouts, but blob has, so it means that:
```
IE::Blob (dims = {1, 3, 2, 2}, Layout = "NCHW") == ov::Tensor(shape = {1, 3, 2, 2})
but:
IE::Blob (dims = {1, 3, 2, 2}, Layout = "NHWC") == ov::Tensor(shape = {1, 2, 2, 3})
```
It is why `get_shape()` returns block dims which are reordered for the blob layout.
This behaviour is aligned with host tensors.